lib/util.js

```diff
@@ -23,6 +23,7 @@ exports.asUserJson = function (user) {
     profileImageUrl: user.profileImageUrl,
     flags: user.flags || {},
     roles: user.roles || [],
+    authorisations: user.authorisations || [],
```

Comment:
we spell it 'authorization' elsewhere in the codebase, so it might be best to be consistent
also, I wonder if we can use the same name on the frontend as client side. a 1:1 relationship seems more intuitive, so these would still be called 'identities' unless there is a specific reason to call them something different (this also matched auth0's user schema)